packages/message/src/index.ts

```diff
@@ -1 +1,26 @@
-export default {};
+export enum PropertyType {
+  Raw,
+  Hash,
+}
+
+export type Property = {
+  type: PropertyType;
+  value: string;
+}
+
+export enum AuthType {
+  Privacy,
+  Package,
+  Local,
+}
+
+export enum KeyType {
+  EDDSA,
+  ECDSA,
+}
+
+export type Signature = {
+  authType: AuthType;
+  keyType: KeyType;
```

Comment:
Key Type is a type to classify the encryption algorithm used, and Auth Type is a type to classify a method for processing a message.